src/server/server.ts

```diff
@@ -48,6 +46,88 @@ export class GraphQLTransportWSServer {
   private wsServer: WebSocket.Server;
   private subscriptionManager: SubscriptionManager;
 
+  // NOTE: The old protocol support should be removed in the future
```

Comment:
you can start write unit test for that 👍 
just get a stream of the old messages and make sure they are converted correctly ;)